htlcswitch/mock.go

```diff
@@ -194,6 +194,7 @@ func initSwitchWithDB(startingHeight uint32, db *channeldb.DB) (*Switch, error)
 	cfg := Config{
 		DB:                   db,
 		FetchAllOpenChannels: db.ChannelStateDB().FetchAllOpenChannels,
+		FetchAllChannels:     db.ChannelStateDB().FetchWaitingCloseChannels,
```

Comment:
I think this only fetches waiting close channels?

Reply:
The comment says it fetches waiting close, pending open, and open channels

Reply:
Will change to FetchAllChannels good catch